pytorch_lightning/trainer/distrib_data_parallel.py

```diff
@@ -377,17 +384,18 @@ def set_nvidia_flags(self, is_slurm_managing_tasks, data_parallel_device_ids):
         # don't make this debug... this is good UX
         rank_zero_info(f'CUDA_VISIBLE_DEVICES: [{os.environ["CUDA_VISIBLE_DEVICES"]}]')
 
-    def set_random_port(self):
+    def set_random_port(self, force=False):
```

Comment:
@jeremyjordan this function is only ever called from ddp on a single node... not distributed